client/driver/qemu.go

```diff
@@ -170,7 +170,10 @@ func (d *QemuDriver) Start(ctx *ExecContext, task *structs.Task) (*StartResponse
 	if d.driverConfig.Accelerator != "" {
 		accelerator = d.driverConfig.Accelerator
 	}
-	// TODO: Check a lower bounds, e.g. the default 128 of Qemu
+
+	if task.Resources.MemoryMB <= 0 || task.Resources.MemoryMB > 4000000 {
```

Comment:
Should be <128 right?

Reply:
Right, sorry overlooked that. I've patched it in the latest commit.